README.md

```diff
@@ -112,4 +112,44 @@ docker run \
     --entrypoint /guardiand \
     ghcr.io/wormhole-foundation/guardiand:latest \
 spy --nodeKey /node.key --spyRPC "[::]:7073" --bootstrap /dns4/${HOST}/udp/8999/quic/p2p/12D3KooWL3XJ9EMCyZvmmGXL2LMiVBtrVa2BuESsJiXkSj7333Jw
+
+# Wormhole-relayer
+
+The Goal is to create a project that is:
```

Comment:
A note since you'll be improving this readme: I think this goals should be in a section "Contributing" or similar. I think the end user wants to see how this repo helps him get VAAs relayed, rather than knowing about this features